MonMaraicher/Features/FarmerDetail/FarmerDetailsView.swift

```diff
@@ -128,22 +114,41 @@ Bienvenue à la ferme de William, où la nature prospère en harmonie. William c
+                .font(.subheadline)
+            Text(viewModel.phoneNumber ?? "")
                 .font(.callout)
-                .padding()
-            Spacer()
-            directionButton
-                .frame(width: 80, height: 35)
-                .background()
-                .clipShape(.capsule)
-                .padding()
+            Divider()
+            Text("Site Web")
+                .foregroundStyle(.secondary)
+                .font(.subheadline)
         }
-        .background(.thinMaterial)
+        .padding()
+        .background(.ultraThinMaterial)
         .clipShape(.rect(cornerRadius: 20))
+        .shadow(radius: 4)
     }
 }
 
 #Preview {
-    FarmerDetailsView(viewModel: FarmerDetailsViewModel(farmer: Farmer(id: 1, name: "chez william", location: .init(latitude: 48.86935, longitude: 2.331314, address: .init(streetNumber: 20, streetName: "rue de la paix", zipCode: 75000, city: "paris")), images: .init(farmer1: "https://plus.unsplash.com/premium_photo-1686529591582-7c53fa833bd9?q=80&w=2940&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D", farmer2: "https://images.unsplash.com/photo-1610348725531-843dff563e2c?q=80&w=2940&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D", farmer3: "https://images.unsplash.com/photo-1542838132-92c53300491e?q=80&w=3174&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D", farmer4: "https://images.unsplash.com/photo-1620706857370-e1b9770e8bb1?q=80&w=3164&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D", farmer5: "https://images.unsplash.com/photo-1488459716781-31db52582fe9?q=80&w=2940&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D", farmer6: "https://images.unsplash.com/photo-1623227866882-c005c26dfe41?w=900&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8MTF8fGZydWl0cyUyMCUyNiUyMGwlQzMlQTlndW1lc3xlbnwwfHwwfHx8MA%3D%3D"))))
+    FarmerDetailsView(viewModel: .init(farmer: .init(id: 1, raisonSociale: "chez william", telephone: nil, telephoneCommerciale: nil, adressesOperateurs: [.init(id: 1, lieu: "20 rue de la paix", codePostal: "75000", ville: "paris", lat: 48.86935, long: 2.331314, typeAdresseOperateurs: ["Lieu de production"]), .init(id: 2, lieu: "300 rue de la paix", codePostal: "75000", ville: "paris", lat: 48.45012, long: 2.354564, typeAdresseOperateurs: ["Magasin"])], productions: [.init(id: 1, code: "", nom: "Citrons"), .init(id: 2, code: "", nom: "Pommes"), .init(id: 3, code: "", nom: "Cerises"), .init(id: 4, code: "", nom: "Légumes frais sous abris avec des céréales"), .init(id: 5, code: "", nom: "Tomates"), .init(id: 6, code: "", nom: "Courgettes")]), address: .init(id: 1, lieu: "20 rue de la paix", codePostal: "75000", ville: "paris", lat: 48.86935, long: 2.331314, typeAdresseOperateurs: ["Siège social"])))
```

Comment:
As you can see, it becomes almost impossible to read. You should find a solution to avoid that :)

Reply:
I think I’m going to create a todo list lol